### ISSUE TITLE
Group normalization support?

### ISSUE THREAD

**Issue by yongtaoge** [osmr/imgclsmob#12]
Is there any plan to make models configurable for batch norm and group norm?

**Comment by yongtaoge** [comment]
Is there any plan to make models configurable for batch norm and group norm?

**Comment by osmr** [comment]
In the near future I have no such plans. 
First of all, because I have no plans to learn such networks in the near future. According to Kaiming He's  paper, the advantage of group normalization is in higher quality for small batches. But few who are trying to train classification models with small batches, due to the fact that you do not achieve the desired quality in that case.
But if you point me to some repository with pre-trained weights for classification models with group normalizations, then I may plan to reproduce these models and convert existing weights.
